docs/Configuration/Dialplan/Contexts-Extensions-and-Priorities.md

```diff
@@ -54,20 +54,20 @@ Within each extension, there must be one or more *priorities*. A priority is sim
   
   
 
-```
```

Comment:
The "tip" just above this line (I can't mark it directly in this review because you didn't change it, should read:

```
!!! tip
    Priority numbers must begin with 1, and must increment sequentially. If Asterisk can't find the next priority number, it will terminate the call. We call this *auto-fallthrough*. Consider the example below:
[//]: # (end-tip)
```